integration-tests/suites/repeated_network_flow.go

```diff
@@ -36,11 +37,17 @@ type RepeatedNetworkFlowTestSuite struct {
 func (s *RepeatedNetworkFlowTestSuite) SetupSuite() {
 	s.StartContainerStats()
 
-	s.Collector().Env["COLLECTOR_CONFIG"] = `{"logLevel":"debug","turnOffScrape":true,"scrapeInterval":` + strconv.Itoa(s.ScrapeInterval) + `}`
-	s.Collector().Env["ROX_AFTERGLOW_PERIOD"] = strconv.Itoa(s.AfterglowPeriod)
-	s.Collector().Env["ROX_ENABLE_AFTERGLOW"] = strconv.FormatBool(s.EnableAfterglow)
+	collectorOptions := common.CollectorStartupOptions{
+		Config: map[string]any{
+			"scrapeInterval": s.ScrapeInterval,
```

Comment:
@JoukoVirtanen is this correct? it seems in this test we're specifically setting a scrape interval, but always turn scraping off.

Reply:
I've tested this locally and the test still passes with `turnScrapeOff=false`. I think this probably better represents what is intended in this test, but could do with some confirmation in case there is some nuance here that I'm missing

Reply:
The scrapeInterval is also the reporting interval. Even if scrape is off the scrapeInterval will be used to determine when connections and endpoint are reported. It is a bit confusing and should be changed. In this case it shouldn't matter if scraping is done or not. In that case network connections will come from NetworkSignalHandler.

Reply:
Thanks! I'll turn off scraping again and fix the conflicts and hopefully we can merge this today